rclcpp/include/rclcpp/executors.hpp

```diff
@@ -104,7 +104,7 @@ template<typename FutureT, typename TimeRepT = int64_t, typename TimeT = std::mi
 rclcpp::FutureReturnCode
 spin_until_future_complete(
   rclcpp::node_interfaces::NodeBaseInterface::SharedPtr node_ptr,
-  const std::shared_future<FutureT> & future,
+  const FutureT & future,
```

Comment:
nit: I know this predates this PR, but I would unify all templates parameter names to either `FutureT` or `FutureResponseT`.
Though I suggested the later, the former sounds nicer :smile:.